src/nplinker/metabolomics/IFileMappingLoader.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC, abstractmethod
+from typing import Dict
+
+
+class IFileMappingLoader(ABC):
+
+    @abstractmethod
+    def mapping(self) -> Dict[int, list[str]]:
```

Comment:
`Dict[]` could be replaced by `dict` in python 3.10

Reply:
Fixed in df7249d0202902f79931d8c61637c1d85e545941